File: Student.cpp
#include<iostream>
#include<String.h>
using namespace std;

class Student
{
    public:
        String fullName;
        int rollNum;
        double semPerentage;
        string collegeName;
        int collegeCode;
        Student()
        {
            cout<<"Student class initialized!!\n";
        }
        ~Student(){}
};

int main()
{
    Student rohith;
    cout<<"Enter your name : ";
    getline(cin, rohith.fullName);
    cout<<"Enter your roll number : ";
    cin>>rohith.rollNum;
    cout<<"Enter your sem percentage : ";
    cin>>rohith.semPerentage;
    cout<<"Enter your college name : ";
    cin>>rohith.collegeName;
    cout<<"Enter your college code : ";
    cin>>rohith.collegeCode;
    cout<<"Name : "<<rohith.fullName<<endl;
    cout<<"RollNum : "<<rohith.rollNum<<endl;
    cout<<"SemPercentage : "<<rohith.semPerentage<<endl;
    cout<<"CollegeName : "<<rohith.collegeName<<endl;
    cout<<"CollegeCode : "<<rohith.collegeCode<<endl;
    return 0;
}

File: student.java
import java.util.Scanner;
public class student
{
    student()
    {
        System.out.println("default constructor called automatically");
    }
    String fullName;
    int rollNum;
    float semper;
    String collegeName;
    int collegeCode;
    
    public static void main(String args[])
    {
        student obj=new student();
        Scanner input= new Scanner(System.in);
        System.out.print("Enter fullname: ");
        obj.fullName= input.next(); 
        System.out.print("Enter your roll no: ");
        obj.rollNum=input.nextInt();
        System.out.print("enter your sem CGPA");
        obj.semper =input.nextFloat();
        System.out.print("Enter your college name: ");
        obj.collegeName=input.next();
        System.out.print("Enter your clg code: ");
        obj.collegeCode=input.nextInt();
    }
}
        

